src/replica.rs

```diff
@@ -422,20 +422,18 @@ impl Replica {
     pub fn expire_tasks(&mut self) -> Result<()> {
         let six_mos_ago = Utc::now() - Duration::days(180);
         let mut ops = Operations::new();
-        self.all_tasks()?
+        self.all_task_data()?
             .drain()
-            .filter(|(_, t)| t.get_status() == Status::Deleted)
```

Comment:
Oh no -- this line should not have been removed! We're now expiring _all_ old tasks, whether deleted or not!